.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 php:
-  - '7.4snapshot'
+  - '7.4'
```

Comment:
Could you please remove 7.4snapshot from the allow_failures matrix

Reply:
I think we can remove the line completely. Right? @sergeyklay

Reply:
Also please check `if [ "$(php-config --vernum)" -ge "70400" ]` condition in this file. Do we still need it?

Reply:
@sergeyklay Upgrade `Development Zephir version` to 7.4 as well?

Reply:
Yes, I think so